jwql/instrument_monitors/nircam_monitors/claw_monitor.py

```diff
@@ -331,8 +331,8 @@ def process(self):
                                  'stddev': float(stddev),
                                  'frac_masked': len(segmap_orig[(segmap_orig != 0) | (dq & 1 != 0)]) / (segmap_orig.shape[0] * segmap_orig.shape[1]),
                                  'skyflat_filename': os.path.basename(self.outfile),
-                                 # 'doy': float(doy),
-                                 # 'total_bkg': float(total_bkg),
+                                 'doy': float(doy),
```

Comment:
Awesome, changes are here from last PR. Thanks!

Reply:
Indeed, and I think the value (in existing entries) will end up being NAN, as the float equivalent to a NULL table value.

Reply:
That's okay, the claw monitor doesn't take too long to run and only has ~1 month of data in it so we can just run it to get all of the fields we need working.